lib/src/Uncertainty/Distribution/TruncatedDistribution.cxx

```diff
@@ -63,22 +63,14 @@ TruncatedDistribution::TruncatedDistribution(const Distribution & distribution,
     const Scalar upperBound,
     const Scalar thresholdRealization)
   : DistributionImplementation()
-  , bounds_(Point(1, lowerBound), Point(1, upperBound))
+  , bounds_(Point(distribution.getDimension(), lowerBound), Point(distribution.getDimension(), upperBound))
```

Comment:
Dont you think it may confuse the user if he does not have to provide explicitely nd bounds ?
Maybe we should only keep only the interval ctor in the long term ?

Reply:
In fact it is useful for many applications where the margins have been standardized, such as in multivariate extreme values analysis where you put standard Frechet margins to your model and study the convergence of symmetrically truncated distributions. For 1D cases, it is also very convenient to specify the bounds as two scalars.